document.go

```diff
@@ -27,3 +33,21 @@ func (client *Client) ServiceApiDocSettingsUpdate(service string, docPath string
 	err := client.Mutate(&m, v, WithName("ServiceApiDocSettingsUpdate"))
 	return &m.Payload.Service, HandleErrors(err, m.Payload.Errors)
 }
+
+type DocumentsConnection struct {
+	Nodes []ServiceDocument
+}
+
+func (client *Client) DocumentSearch(searchTerm string) (*DocumentsConnection, error) {
+	var q struct {
+		Account struct {
+			Documents DocumentsConnection `graphql:"documents(searchTerm: $searchTerm)"`
+		}
+	}
+
+	v := PayloadVariables{
+		"searchTerm": &searchTerm,
+	}
+	err := client.Query(&q, v, WithName("DocumentSearch"))
```

Comment:
This is likely missing automatic pagination (unless we have a restriction of no more then 100 documents on the ruby side)